build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+tagCount=$(git tag -l --points-at HEAD | wc -l)
+if [ $tagCount -gt 0 ]; then
+  echo "tagged so deploy"
+  ant -f ./client-libraries/java/rest-client/build.xml -lib ./client-libraries/java/rest-client/lib deploy
+else
+  echo "not tagged so stage"
```

Comment:
would be cleaner to no op if it's a pull request, e.g.
"${TRAVIS_PULL_REQUEST}" == "false"